docs/pages/access-controls/guides/webauthn.mdx

```diff
@@ -129,10 +128,11 @@ Update the configuration:
 $ tctl create -f cap.yaml
 # cluster auth preference has been updated
 ```
-
 </ScopedBlock>
 
-You will need to include the following configuration fields.
+<Admonition type="tip" title="U2F">
+  Starting on Teleport v10, WebAuthn replaces U2F. See the [U2F](#u2f) section.
```

Comment:
To prevent the reader from jumping around this page, I think we can:
- Turn this Admonition into a Details box called "Using U2F?" or something similar
- Move the contents of the U2F section into the new Details box

Reply:
Changed to Details, but kept the separate U2F section. I didn't like the way text flows from U2F to the rest of this section, as users are not really supposed to follow both.